packages/@sfx/core/test/unit/common/core.test.ts

```diff
@@ -0,0 +1,84 @@
+import { expect, sinon, spy, stub } from '../../utils';
+import Core from '../../../src/core';
+import * as CoreUtils from '../../../src/utils/core';
+
+describe('Core', () => {
+  let core: Core;
+
+  beforeEach(() => {
+    core = new Core();
+  });
+
+  describe('constructor()', () => {
+    it('should create an empty null-prototype plugins registry object', () => {
+      expect(core.plugins).to.be.empty;
+      expect(Object.getPrototypeOf(core.plugins)).to.be.null;
+    });
+  });
+
+  describe('register()', () => {
+    let calculateMissingDependencies = stub(CoreUtils, 'calculateMissingDependencies').returns(['x']);
+    let registerPlugins = stub(CoreUtils, 'registerPlugins');
+    let initPlugins = stub(CoreUtils, 'initPlugins');
+    let readyPlugins = stub(CoreUtils, 'readyPlugins');
```

Comment:
The above 4 stubs seem redundant as this is also being done in the `beforeEach` step. Could you just initialize these first without the stub?

Reply:
Looks like this was a leftover from an incomplete refactor. You're right, these should be removed and the ones in the `beforeEach` should stay.